cvat/apps/lambda_manager/views.py

```diff
@@ -247,6 +247,22 @@ def parse_attributes(attrs_spec):
         self.help_message = meta_anno.get("help_message", "")
         self.gateway = gateway
 
+        if "supported_shape_types" in meta_anno:
```

Comment:
Backward compatibility is a good thing, but I believe we anyway should update existing tracker specs to come to unified interface somewhere in the future (I mean adding "supported_shape_types" to all trackers)

Reply:
I don't really see a reason to do that. The compatibility code is simple enough that I think we can maintain it indefinitely.